apps/dashboard/src/components/subscribers/locale-select.tsx

```diff
@@ -1,71 +1,111 @@
 import { locales } from '@/utils/locales';
-import { RiEarthLine } from 'react-icons/ri';
-import { CountryFlags } from '../icons/country-flags';
-import { Select, SelectContent, SelectItem, SelectTrigger, SelectValue } from '../primitives/select';
+import { cn } from '@/utils/ui';
+import { RiArrowDownSLine, RiCheckLine, RiEarthLine } from 'react-icons/ri';
+import { type Country } from 'react-phone-number-input';
+import flags from 'react-phone-number-input/flags';
+import { Button } from '../primitives/button';
+import { Command, CommandEmpty, CommandGroup, CommandInput, CommandItem, CommandList } from '../primitives/command';
+import { Popover, PopoverContent, PopoverTrigger } from '../primitives/popover';
+import { ScrollArea } from '../primitives/scroll-area';
 import TruncatedText from '../truncated-text';
+import { useState } from 'react';
 
 export function LocaleSelect({
   value,
-  defaultOption,
+  onChange,
   disabled,
-  onValueChange,
   readOnly,
-  required,
 }: {
   value?: string;
-  defaultOption?: string;
-  size?: 'sm' | 'md';
   disabled?: boolean;
   readOnly?: boolean;
-  required?: boolean;
-  onValueChange: (val: string) => void;
+  onChange: (val: string) => void;
 }) {
+  const [open, setOpen] = useState(false);
```

Comment:
the list is not scrolling up when the search is performed:
 

https://github.com/user-attachments/assets/3cf41700-14ce-4471-921a-00f5be1041ef

Reply:
same thing in the timezone

Reply:
It's a bug in the cmdk library😓 
will try some hacks

Reply:
:/